src/LineageTree/lineageTree.py

```diff
@@ -470,8 +470,7 @@ def to_treex(self, sampling=1, start=0, finish=10000, many=True):
 
         """
         from treex.tree import Tree
-        from warnings import warn
-
+        from warnings importprint("Peos2")
```

Comment:
what's that?

Reply:
Oops I forgot about that, i use it for testing